slack-base/src/main/java/com/hubspot/slack/client/models/users/ProfileFieldIF.java

```diff
@@ -0,0 +1,14 @@
+package com.hubspot.slack.client.models.users;
+
+import org.immutables.value.Value.Immutable;
+
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Immutable
+@HubSpotStyle
+public interface ProfileFieldIF {
+
+  String getValue();
+
+  String getAlt();
```

Comment:
Sounds like the `alt` field is [optional](https://api.slack.com/methods/users.profile.set):
> The `value` of a field is what should be displayed, unless the `alt` key is also present, in which case that is displayed instead.